.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+<!-- Please update this template with something that matches your PR -->
+| Q             | A
+| ------------- | ---
+| Bug fix?      | yes/no
+| New feature?  | yes/no
+| BC breaks?    | no
+| Deprecations? | yes/no
+| Tests pass?   | yes
+| Fixed tickets | Fix #... <!-- prefix each issue number with "Fix #", if any -->
+| License       | MIT
+| Doc PR        | api-platform/docs#...
+
+<!-- Bug fixes should be based against the current stable version branch, master is for new features only -->
```

Comment:
Please add comments about:
- BC breaks (for PHP)
- adding some tests
- updating the CHANGELOG.md